### PR TITLE
EWSv2: timezone fix

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39240

## Description
Fix the docker with the TZ issue.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by glicht** [comment]
TDD: failed unit test before docker update: https://app.circleci.com/pipelines/github/demisto/content/100847/workflows/dc788608-dbf3-4a33-9253-23287523e545/jobs/398640

File: Packs/EWS/ReleaseNotes/1_8_27.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### EWS v2
+- Fixed an issue where ***fetch-incidents*** sometimes failed with an `UnknownTimeZoneError`.
+- Updated the Docker image to: *demisto/py-ews:1.0.0.22402*.
```
Comment (anara123) [inline Packs/EWS/ReleaseNotes/1_8_27.md:5]:
```suggestion
- Updated the Docker image to: *demisto/py-ews:1.0.0.23126*.
```

**Review by anara123 — APPROVED** [review]

File: Packs/EWS/ReleaseNotes/1_8_27.md (hunk 1 above)
Comment (glicht) [reply Packs/EWS/ReleaseNotes/1_8_27.md:5]: Good catch

Comment (glicht) [reply Packs/EWS/ReleaseNotes/1_8_27.md]: pushed a commit and also updated the fix line with the version that the issue was introduced in.

**Pull request merged**